src/routes/metadata.ts

```diff
@@ -4,4 +4,34 @@ import * as db from '../tools/db';
 // for location, categories, etc
 
 export const register = (app: express.Application) => {
+
+    //GET ProductCategories
+    app.get('/api/v1/prodcats',(req, res) =>{
```

Comment:
```suggestion
    app.get('/api/v1/prodcast',(req: Request, res: Response) => {
```

You misspelled podcast and please add the type annoations

Reply:
I think it's supposed to be prodcats, as in product categories, so we're good here